src/os/bluestore/BlueStore.cc

```diff
@@ -1851,8 +1846,13 @@ void BlueStore::BufferSpace::read(
   cache->logger->inc(l_bluestore_buffer_miss_bytes, miss_bytes);
 }
 
-void BlueStore::BufferSpace::_finish_write(BufferCacheShard* cache, uint64_t seq)
+void BlueStore::BufferSpace::_finish_write(BufferCacheShard* cache, uint64_t offset, uint64_t seq)
```

Comment:
`offset` looks like a debug addition, to remove?

Reply:
yes, will be removed, it is a spillover of previous failed implementations